solar/solar/interfaces/orm.py

```diff
@@ -477,6 +483,40 @@ def add_input(self, name, schema, value):
         self.inputs.add(input)
 
 
+class DBEvent(DBObject):
+
+    __metaclass__ = DBObjectMeta
+
+    _collection = base.BaseGraphDB.COLLECTIONS.events
+
+    id = db_field(is_primary=True)
+    parent = db_field(schema='str!')
+    parent_action = db_field(schema='str!')
+    etype = db_field('str!')
+    state = db_field('str')
+    child = db_field('str')
+    child_action = db_field('str')
+
+    @classmethod
+    def load_list(cls, parent):
+        rs = db.all(collection=cls._collection.name + ':' + parent)
+        return [cls(**r.properties) for r in rs]
+
+    @classmethod
+    def delete_list(cls, parent):
+        db.delete('*', collection=cls._collection.name + ':' + parent)
```

Comment:
I would make a `delete_all` method in `db` not to add strange things as `*` here. Also, same remark about `_collection.name + ':' + parent` as above.